components/General/IconHeader.tsx

```diff
@@ -20,11 +20,17 @@ export default function IconHeader({ title, description }) {
           alt="Icon"
           className="mr-2 h-8 w-8"
         />
-        <Text className="text-xl font-bold">{title}</Text>
+        <Text
+          className="text-h2-sm-medium"
+          style={{ color: colors.textTitleGrayscale }}
+        >
+          {title}
+        </Text>
       </View>
-      <Text className="font-montserrat px-6 pb-4 pl-6 text-sm">
+      <Text className="text-h2-sm-medium px-6 pb-4 pl-6 text-sm">
         {description}
       </Text>
     </>
   );
-}
+};
+export default IconHeader;
```

Comment:
For next time: Always use inline named exports, not default exports. Only use default exports if the component is imported without `{}` in components that you don't intend to edit, as removing the default export would break those imports. See https://erasmusegalitarian.github.io/educado-docs/handbook/mobile/standards/#use-inline-exports-except-in-the-app-directory